src/util/strconv.h

```diff
@@ -20,6 +21,9 @@
 // Case sensitive string equal
 #define STR_EQ(str, len, other) (len == strlen(other) && !strncmp(str, other, len))
 
+// Threshold for Small String Optimization (SSO)
+#define SSO_MAX_LENGTH 16
```

Comment:
Any reason for this threshold? Can we increase it? e.g., what about 32, 64, or 128?

Reply:
Because the average length of an English word is less than 16. But tokens can be larger than words, so I changed it now to 128.